environments/hca/terraform/dagster.tf

```diff
@@ -0,0 +1,40 @@
+resource "google_project_iam_custom_role" "argo_access" {
+  role_id     = "argoworkflows.user"
```

Comment:
❓ Should we delete the `DagsterArgoRole` we were debugging with?

Reply:
yeah, i think so. the id i'm using here more closely aligns with the naming scheme of other iam roles.